src/main/java/com/commercetools/pingpong/controller/GameController.java

```diff
@@ -21,21 +20,40 @@
     public String getPlayers(Model model) {
         model.addAttribute("rightPlayer", gameService.getRightPlayer());
         model.addAttribute("leftPlayer", gameService.getLeftPlayer());
+        model.addAttribute("servingPlayer", servingPlayer());
         return "current-game";
     }
 
-    @RequestMapping(path = "/score")
-    public void updateScorePlusOne(@RequestParam(value="player", required = false) String player, Model model) {
+
+        public  String servingPlayer() {
+        if (gameService.getRightPlayerServe()) {
+            return "Serving Player: Philipp";
+
+        } else if (gameService.getLeftPlayerServe()) {
+            return "Serving Player: Nicola";
+    // FIXME: names will be changed later
+        }
+        return "Um die Angabe";
+    }
+
+    @RequestMapping(path = "/score", method = RequestMethod.POST)
+    @ResponseBody
+    public String updateScorePlusOne(@RequestParam(value="player", required = false) String player, Model model) {
         gameService.updateScore(new Message(player, 1));
+        return "Score updated";
     }
 
     @RequestMapping(path = "/delete")
-    public void updateScoreDelete(@RequestParam(value="player", required = false) String player, Model model) {
+    @ResponseBody
+    public String updateScoreDelete(@RequestParam(value="player", required = false) String player, Model model) {
         gameService.updateScore(new Message(player, 2));
+        return "Point deleted";
     }
 
     @RequestMapping(path = "/reset")
-    public void updateScoreReset(@RequestParam(value="player", required = false) String player, Model model) {
+    @ResponseBody
+    public String updateScoreReset(@RequestParam(value="player", required = false) String player, Model model) {
         gameService.updateScore(new Message(player, 3));
+        return "Score reseted";
```

Comment:
https://www.verbix.com/webverbix/English/reset.html 🤓

Reply:
hahaha yes, i saw that yesterday already 🤓 
4826e16e5763ae7cbe11b2983fe809e1721096c5